lite/examples/model_personalization/converter/tfltransfer/heads/keras_model_head.py

```diff
@@ -121,19 +125,20 @@ def train(self, bottleneck, labels, scope='head'):
     bottleneck_names = [
         input_def.name
         for key, input_def in self._eval_signature.inputs.items()
-        if key.endswith('_input')
+        if 'input' in key
```

Comment:
Can we make sure that the 'input' and 'target' are not in the middle of the key names?

Reply:
Sorry for the delay, implemented this.